diff --git a/Dockerfile.dapper b/Dockerfile.dapper
index e5149cb..8a72234 100644
--- a/Dockerfile.dapper
+++ b/Dockerfile.dapper
@@ -1,6 +1,6 @@
 FROM registry.suse.com/bci/bci-base:15.4
 
-ARG DAPPER_HOST_ARCH=amd64
+ARG DAPPER_HOST_ARCH=ppc64le
 ARG http_proxy
 ARG https_proxy
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
@@ -30,7 +30,7 @@ RUN if [ ${ARCH} == "s390x" ]; then \
     fi
     
 # Install Go & tools
-ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH_s390x=s390x GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
+ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH_s390x=s390x GOLANG_ARCH_ppc64le=ppc64le GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 RUN wget -O - https://storage.googleapis.com/golang/go1.17.10.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
     go get -u golang.org/x/lint/golint
@@ -40,6 +40,7 @@ RUN wget -O - https://storage.googleapis.com/golang/go1.17.10.linux-${!GOLANG_AR
 ENV GRPC_HEALTH_PROBE_amd64=https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-amd64 \
     GRPC_HEALTH_PROBE_arm64=https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-arm64 \
     GRPC_HEALTH_PROBE_s390x=https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-s390x \
+    GRPC_HEALTH_PROBE_ppc64le=https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-ppc64le \
 		GRPC_HEALTH_PROBE=GRPC_HEALTH_PROBE_${ARCH}
 
 RUN wget ${!GRPC_HEALTH_PROBE} -O /usr/local/bin/grpc_health_probe && \
@@ -49,6 +50,7 @@ RUN wget ${!GRPC_HEALTH_PROBE} -O /usr/local/bin/grpc_health_probe && \
 ENV PROTOC_amd64=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-x86_64.zip \
 	PROTOC_arm64=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-aarch_64.zip \
 	PROTOC_s390x=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-s390_64.zip \
+        PROTOC_ppc64le=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VER}/protoc-${PROTOBUF_VER}-linux-ppcle_64.zip \
 		PROTOC=PROTOC_${ARCH}
 
 RUN cd /usr/src && \
diff --git a/package/Dockerfile b/package/Dockerfile
index 876bbaa..df708cb 100644
--- a/package/Dockerfile
+++ b/package/Dockerfile
@@ -1,6 +1,6 @@
 FROM registry.suse.com/bci/bci-base:15.4
 
-ARG ARCH=amd64
+ARG ARCH=ppc64le
 
 # Based on https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/blob/master/deploy/docker/x86_64/Dockerfile
 FROM registry.suse.com/suse/sle15:15.4 AS build
diff --git a/scripts/package b/scripts/package
index 3831336..c650c34 100755
--- a/scripts/package
+++ b/scripts/package
@@ -14,6 +14,9 @@ case $(uname -m) in
   x86_64)
     ARCH=amd64
     ;;
+  ppc64 | ppc64le)
+    ARCH=ppc64le
+    ;;
   s390x)
     ARCH=s390x
     ;;
